packages/lib/gpc/src/gpcTypes.ts

```diff
@@ -359,3 +368,57 @@ export type GPCRevealedClaims = {
    */
   watermark?: PODValue;
 };
+
+/**
+ * General-purpose circuit requirements for a given proof configuration.
```

Comment:
"for a given proof."  Requirements are derived from both "configuration" and "inputs", which are distinct concepts at this level.